app/controllers/admin/roles_controller.rb

```diff
@@ -8,22 +8,40 @@ class RolesController < Admin::BaseController
 
     def index
       @roles = Role.where(resource: @conference)
+      tracks = @conference.program.tracks.where.not(submitter: nil)
+      @roles += Role.where(resource: tracks)
       authorize! :index, @role
     end
 
     def show
+      @url = if @track
+               toggle_user_track_admin_conference_role_path(@conference.short_title, @role.name, @track.name.tr(' ', '_'))
```

Comment:
Instead of changing the track.name, it would make sense to add a short_name like conference has, and change how we navigate to tracks all together, and base it on track.short_name instead of track.id